src/server/identity/server/api_server.go

```diff
@@ -57,6 +58,21 @@ func (a *apiServer) EnvBootstrap(ctx context.Context) error {
 		if err := yaml.Unmarshal([]byte(a.env.Config.IdentityConnectors), &connectors); err != nil {
 			return errors.Wrapf(err, "unmarshal IdentityConnectors: %q", a.env.Config.IdentityConfig)
 		}
+		// IDPConnector.Config can't be unmarshalled to directly, so it has to be manually unmarshalled.
+		var tmpConnectors []map[string]interface{}
```

Comment:
I think it's worth exploring if we can avoid this entirely by unmarshalling directly into map[string]interface{}